files/default/audit_report.rb

```diff
@@ -6,27 +6,44 @@ class Handler
     # Creates a compliance audit report
     class AuditReport < ::Chef::Handler
       def report
-        reporter = node['audit']['collector']
+        # ensure reporters is array (see libraries/helper.rb)
+        reporters = handle_reporters(node['audit']['collector'])
+
+        # collect attribute values
         server = node['audit']['server']
         user = node['audit']['owner']
         token = node['audit']['token']
         refresh_token = node['audit']['refresh_token']
         interval = node['audit']['interval']
         interval_enabled = node['audit']['interval']['enabled']
         interval_time = node['audit']['interval']['time']
-        report_file = node['audit']['output']
         profiles = node['audit']['profiles']
+        quiet = node['audit']['quiet']
+
+        # create a file with a timestamp to
+        create_timestamp_file if interval_enabled
 
+        # load inspec, supermarket bundle and compliance bundle
         load_needed_dependencies
 
-        # ensure authentication for Chef Compliance is in place
-        login_to_compliance(server, user, token, refresh_token) if reporter == 'chef-compliance'
+        # iterate through reporters
+        reporters.each do |reporter|
```

Comment:
it would be perfect to iterate only over `send_report(reporter, server, user, profiles)`, but we need `json-min` for Chef Compliance ... so it looks like we need to iterate over the scan as well for now